spec/factories/learning_hour_types.rb

```diff
@@ -0,0 +1,8 @@
+FactoryBot.define do
+  factory :learning_hour_type do
+    casa_org { CasaOrg.first || create(:casa_org) }
+    name { Faker::Book.genre }
```

Comment:
suggest `sequence(:name) { "Learning Hour Type #{n}" }` to avoid uniqueness issues in specs